test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -390,6 +387,118 @@ std::string unitTestNameFromLaunchArguments(const winrt::ILaunchActivatedEventAr
     return unitTestName;
 }
 
+constexpr auto timeout{ std::chrono::seconds(5) };
+
+winrt::ToastProgressResult progressResultOperationHelper(winrt::IAsyncOperation<winrt::ToastProgressResult> progressResultOperation)
+{
+    if (progressResultOperation.wait_for(timeout) != winrt::AsyncStatus::Completed)
+    {
+        progressResultOperation.Cancel();
+        return winrt::ToastProgressResult::Failed; // timed out or failed
+    }
+
+    return progressResultOperation.GetResults();
+}
+
+bool VerifyUpdateToastProgressDataUsingValidTagAndValidGroup()
+{
+    winrt::ToastProgressData progressData{};
+    progressData.Status(L"SomeStatus");
+    progressData.Title(L"SomeTitle");
+    progressData.Value(0.14);
+    progressData.ValueStringOverride(L"14%");
+
+    // TODO: PoastToast 
+    auto progressResultOperation = winrt::ToastNotificationManager::Default().UpdateToastProgressDataAsync(progressData, L"Tag", L"Group");
+    auto progressResult = progressResultOperationHelper(progressResultOperation);
+
+    if (progressResult == winrt::ToastProgressResult::Succeeded)
```

Comment:
You can condense the last lines into:

`return progressResult == winrt::ToastProgressResult::Succeeded;`

Reply:
Same comment applies to other tests.